deploy-cloudgov-buildpack.sh

```diff
@@ -0,0 +1,155 @@
+#!/bin/sh
+#
+# This script will attempt to create the services required
+# and then launch everything.
+#
+
+# The deployment strategy you wish to employ ( rolling update or setting up a new environment)
+DEPLOY_STRATEGY=${1}
+
+# The environment in which you want to execute these commands
+DEPLOY_ENV=${2}
+
+#The application name  defined via the manifest yml for the backend
+CGHOSTNAME_BACKEND=${3}
+
+#The application name  defined via the manifest yml for the frontend
+CGHOSTNAME_FRONTEND=${4}
+
+
+#The Github Branch triggered to execure this script if triggered in circleci
+CIRCLE_BRANCH=${7}
+
+echo DEPLOY_STRATEGY: $DEPLOY_STRATEGY
+echo DEPLOY_ENV=$DEPLOY_ENV
+echo BACKEND_HOST: $CGHOSTNAME_BACKEND
+echo FRONTEND_HOST: $CGHOSTNAME_FRONTEND
+echo CIRCLE_BRANCH=$CIRCLE_BRANCH
+
+
+
+# Tell people where to go
+echo
+echo
+echo "to log into the site, you will want to go to https://${CGHOSTNAME_FRONTEND}.app.cloud.gov/"
+echo 'Have fun!'
+
+# Performs a normal deployment unless rolling is specified in the fucntion call
+update_frontend()
+{
+  cd tdrs-frontend && npm run dist && cd ..
+	if [ "$1" = "rolling" ] ; then
+		# Do a zero downtime deploy.  This requires enough memory for
+		# two apps to exist in the org/space at one time.
+		cf push $CGHOSTNAME_FRONTEND --no-route -f tdrs-frontend/manifest.yml --strategy rolling || exit 1
+	else
+		cf push $CGHOSTNAME_FRONTEND --no-route -f tdrs-frontend/manifest.yml
+	fi
+	cf map-route $CGHOSTNAME_FRONTEND app.cloud.gov --hostname "${CGHOSTNAME_FRONTEND}"
+}
+
+# Performs a normal deployment unless rolling is specified in the fucntion call
+update_backend()
+{
+	if [ "$1" = "rolling" ] ; then
+		# Do a zero downtime deploy.  This requires enough memory for
+		# two apps to exist in the org/space at one time.
+		cf push $CGHOSTNAME_BACKEND --no-route -f tdrs-backend/manifest.yml  --strategy rolling || exit 1
+
+	else
+		cf push $CGHOSTNAME_BACKEND --no-route -f tdrs-backend/manifest.yml
+		# set up JWT key if needed
+		if cf e $CGHOSTNAME_BACKEND | grep -q JWT_KEY ; then
+		   echo jwt cert already created
+		else
```

Comment:
This is the file I will be running, the others are incomplete breakdowns of the logic in here.
Planning to add to scripts dir
+ deploy
+ deploy-frontend-update
+ deploy-backend-
+ deploy-setup
+ deploy-setup-frontend
+ deploy-setup-backend